src/monkey_patch_headers.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+const http = require('http');
+
+function findOutHedersSymbol() {
+    const x = new http.OutgoingMessage();
+    const s = Object.getOwnPropertySymbols(x);
+    assert(typeof s[0] === 'symbol'
+        && s[0].toString() === 'Symbol(outHeadersKey)');
+    return s[0];
+}
+
+function monkeyPatchHeaders(options) {
```

Comment:
This would deserve a comment, explain how it works, what it does and what is it good for